### PR TITLE
Adding in organic vs synth and delay times

### PR DESCRIPTION
**Pull request by wanderingweights**

### CODE REVIEW

File: validator/core/cycle.py [hunk 1]
```diff
@@ -175,7 +176,8 @@ async def _let_miners_know_to_start_training(
         hours_to_complete=task.hours_to_complete,
     )
 
-    logger.info(f"We are telling miners to start training there are {len(nodes)}")
+    logger.info(
+        f"We are telling miners to start training there are {len(nodes)}")
```
Comment (namoray) [inline validator/core/cycle.py:180]: , missin i reckon

File: validator/evaluation/docker_evaluation.py [hunk 2]
```diff
@@ -63,12 +63,12 @@ async def run_evaluation_docker(
         )
         # NOTE: replace with asyncio.to_thread
 
-        log_thread = threading.Thread(target=stream_logs, args=(container,))
-        log_thread.start()
+#        log_thread = threading.Thread(target=stream_logs, args=(container,))
```
Comment (namoray) [inline validator/evaluation/docker_evaluation.py:66]: intended to stay commented out? If so, can we delete? If not, perhaps a # TODO: or # NOTE: to remove comment

File: validator/core/constants.py [hunk 3]
```diff
@@ -67,15 +66,19 @@
 MAX_IDEAL_NUM_MINERS_IN_POOL = 20
 MIN_COMPETITION_HOURS = 1
 MAX_COMPETITION_HOURS = 5
```
Comment (namoray) [inline validator/core/constants.py:68]: 3 for now?

File: validator/evaluation/docker_evaluation.py (hunk 2 above)
Comment (wanderingweights) [reply validator/evaluation/docker_evaluation.py:66]: Ah this is me playin with things to avoid getting stuck with loading a model

**Pull request closed without merging**